start_miner.sh

```diff
@@ -22,5 +22,4 @@ pm2 start neurons/miner.py --name bitmind_miner -- \
   --wallet.name $WALLET_NAME \
   --wallet.hotkey $WALLET_HOTKEY \
   --axon.port $MINER_AXON_PORT \
-  --blacklist.force_validator_permit $BLACKLIST_FORCE_VALIDATOR_PERMIT \
-  --logging.info
+  --blacklist.force_validator_permit $BLACKLIST_FORCE_VALIDATOR_PERMIT
```

Comment:
enabling info logging by default in individual scripts (we always want this on for visibility in w&b)